test/tools/ossfuzz/yulProto_diff_ossfuzz.cpp

```diff
@@ -63,10 +63,14 @@ DEFINE_PROTO_FUZZER(Function const& _input)
 		// Parse protobuf mutated YUL code
 		if (!stack.parseAndAnalyze("source", yul_source) || !stack.parserResult()->code ||
 			!stack.parserResult()->analysisInfo)
+		{
+			YulStringRepository::instance().reset();
```

Comment:
Wouldn't it be better to just reset at the start?

Reply:
Indeed :-)